src/restapi/fDisconnect.ts

```diff
@@ -14,6 +15,16 @@ export async function fDisconnect(req: Request, res: Response): Promise<void> {
         botAccessToken: context.bot_access_token,
         mattermostSiteUrl: context.mattermost_site_url,
     };
+
+    // get the saved service account config zendesk access_token
+    const config = await newConfigStore(context.bot_access_token, context.mattermost_site_url).getValues();
+    const configOauthToken = config.zd_oauth_access_token;
+    const text = 'This mattermost account is connected via oauth2 to Zendesk for subscription functionality.  The account cannot be disconnected until the access token in the configuration is updated to a new user access token.';
```

Comment:
```suggestion
    const text = 'This mattermost account is connected via oauth2 to Zendesk for subscription functionality. The account cannot be disconnected until the access token in the configuration is updated to a new user access token.';
```

Reply:
My comment was just about the double white space not more.

Wtr i18n: Either the i18n files are served via static assets or just places next to the code. I'm a bit concerned about i18n in AWS as loading the files take a lot of time and memory. I suppose it's fine for small apps. Most importantly, only the needed language should be loaded, not all of them.

Reply:
Double space removed.